pkg/monitor/monitorapi/identification.go

```diff
@@ -0,0 +1,63 @@
+package monitorapi
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func E2ETestLocator(testName string) string {
+	return fmt.Sprintf("e2e-test/%q", testName)
```

Comment:
this will produce a weird string, no? `e2e-test/"foo bar`

Reply:
> this will produce a weird string, no? `e2e-test/"foo bar"` .. do we want those quotes?

we need them for e2e tests since those can contain slashes.

Reply:
I removed the others.